include/srdfdom/model.h

```diff
@@ -195,6 +195,19 @@ class Model
     std::string name_;
   };
 
+  // Some joints may have additional numerical properties.
```

Comment:
I'm not happy you want to represent this as a `std::string` then. A number of other properties in this header already use `double`.

Can you really make a case for `string` in favor of `double`?
Casting double to int is much cheaper than atoi/atof and I'm not convinced you need to represent 64-bit integer values double can't accurately represent.

Reply:
Yea pls provide example as comment what a non-numeric joint value is.

Reply:
@DLu your example is great, but it uses no numerical property at all. So please just drop the "numerical" here then.

Reply:
Good catch